### PR TITLE
Apply go fmt

### PR DESCRIPTION
**Pull request by zhouhao3**
## Which problem is this PR solving?
- 

## Short description of the changes
- 
I found that [ALL_SRC](https://github.com/jaegertracing/jaeger/blob/master/Makefile#L6-L21) does not include some folders in the Makefile, which results in the format of some files that cannot be detected by make fmt. Not sure if this is acceptable.

Signed-off-by: Zhou Hao <zhouhao@cn.fujitsu.com>

### CODE REVIEW

File: thrift-gen/agent/agent-consts.go [hunk 1]
```diff
@@ -3,15 +3,14 @@
...
+import (
 	"bytes"
 	"context"
-	"reflect"
 	"fmt"
 	"github.com/apache/thrift/lib/go/thrift"
 	"github.com/jaegertracing/jaeger/thrift-gen/jaeger"
 	"github.com/jaegertracing/jaeger/thrift-gen/zipkincore"
-
+	"reflect"
```
Comment (jpkrohling) [inline thrift-gen/agent/agent-consts.go:13]: Something went wrong here.

File: thrift-gen/agent/agent.go [hunk 2]
```diff
@@ -3,15 +3,14 @@
...
+import (
 	"bytes"
 	"context"
-	"reflect"
 	"fmt"
 	"github.com/apache/thrift/lib/go/thrift"
 	"github.com/jaegertracing/jaeger/thrift-gen/jaeger"
 	"github.com/jaegertracing/jaeger/thrift-gen/zipkincore"
-
+	"reflect"
```
Comment (jpkrohling) [inline thrift-gen/agent/agent.go:13]: And here :)

**Review by jpkrohling — COMMENTED** [review]
Shouldn't this PR include some change to the Makefile to include the new paths? I would also like to leave the generated files alone, or to apply a `make fmt` right after generating them. Otherwise, they'll be reverted whenever they are re-generated.

**Comment by yurishkuro** [comment]
We do not want to format or lint auto-generated files.

**Comment by zhouhao3** [comment]
updated, PTAL.

**Comment by yurishkuro** [comment]
This still includes mocks, which are auto-generated.

**Comment by zhouhao3** [comment]
Done.

**Pull request merged**